packages/dropdowns/src/elements/Trigger/Trigger.tsx

```diff
@@ -39,25 +40,48 @@ const Trigger: React.FunctionComponent<ITriggerProps> = ({ children, refKey, ...
     }
 
     previousIsOpenRef.current = isOpen;
-  }, [isOpen]);
+
+    if (hasMenuRef.current === false) {
+      hasMenuRef.current = true;
+    }
```

Comment:
Does the `hasMenuRef` logic have anything to do with the `isOpen` state? This may need to be moved to its own `useEffect`.

Reply:
Ah yes, good catch. Since `hasMenuRef` has nothing to do with `isOpen`, I've moved it into its own `useEffect`.